build.gradle

```diff
@@ -49,5 +49,19 @@ allprojects {
         options.compilerArgs += ['-Werror']
     }
 
+    tasks.withType(Test) {
+        useJUnitPlatform {
+            includeEngines 'junit-jupiter'
+        }
+        maxParallelForks Runtime.getRuntime().availableProcessors()
+        testLogging {
+            events "passed", "skipped", "failed"
+            exceptionFormat "full"
+            showCauses true
+            showExceptions true
+            showStackTraces true
+        }
+    }
```

Comment:
huh, I'd have expected this to be handled out of the box by baseline

Reply:
I can rip this out, I had copied over from tritium to see all the tests running.